scanner/matcher/updater/vuln/updater.go

```diff
@@ -483,7 +488,7 @@ func (u *Updater) runMultiBundleUpdate(ctx context.Context) error {
 		zlog.Info(ctx).Msg("starting bundle update")
 		if err := u.updateBundle(ctx, bundleF, zipTime, prevTime); err != nil {
```

Comment:
TODO: I think I will actually add a `bool` return for this, too, and block the GC from running if this function never returns `true`

Reply:
may not do this to keep things simple